src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,11 +1,27 @@
 package core.basesyntax;
 
 public class UserService {
+
     public void registerUser(User user) {
 
+        try {
+
+            PasswordValidator validator = new PasswordValidator();
+            UserService userService = new UserService();
```

Comment:
do we need to creat an object of UserService?

Reply:
done!